lib/events/roomcreateevent.h

```diff
@@ -10,6 +10,11 @@ class RoomCreateEvent : public StateEventBase {
 public:
     DEFINE_EVENT_TYPEID("m.room.create", RoomCreateEvent)
 
+    enum RoomType {
+        Space,
+        Undefined,
+    };
```

Comment:
```suggestion
    };
    Q_ENUM(RoomType)
```
? Maybe even move the enum out of `RoomCreateEvent` as it might be useful/needed in other contexts.